Mobile Buy SDK/Mobile Buy SDK/Models/BUYCheckout.h

```diff
@@ -149,11 +149,6 @@
 @property (nonatomic, strong) BUYShippingRate *shippingRate;
 
 /**
- *  Shipping rate identifier
- */
-@property (nonatomic, readonly) NSString *shippingRateId;
```

Comment:
Added it back and marked as deprecated along with the implementation of a setter.